bazel/example/main.cc

```diff
@@ -10,5 +11,9 @@ int main(int argc, char* argv[]) {
 
   LOG(INFO) << "Hello, world!";
 
+  // glog/stl_logging.h allows logging STL containers.
+  std::vector<int> x {1, 2, 3};
```

Comment:
This will not compile in pre C++11 mode. Is this intentional?

Reply:
No, good point.